src/components/TodoInput.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+function TodoInput({ todoList, setTodoList }) {
+  const [todo, setTodo] = useState('');
+  const onHandleInputChange = (e) => {
```

Comment:
보통 `handle + 대상 + 동작`의 네이밍이 일반적입니다!
```suggestion
  const handleInputChange = (e) => {
```